tests/test_minimal/test_tools/test_nwb_helpers.py

```diff
@@ -51,3 +51,11 @@ def test_metadata_integrity(self):
         assert metadata["NWBFile"]["session_description"] == session_description
         assert metadata["NWBFile"]["identifier"] == identifier
         assert metadata["NWBFile"]["session_start_time"] == session_start_time
+
+    def test_make_nwbfile_from_metadata_called_twice(self):
```

Comment:
The actual test here is for non-modification of the `metadata`

Can you simplify the test by simply asserting that output dict is identically equal to output dict?